contrib/ext_editor.lua

```diff
@@ -409,27 +410,11 @@ local box1 = dt.new_widget("box") {
   button_update_list
   }
 
+
+-- install module in lighttable and darkroom ----------------------------------
 table.insert(ee.widgets, combobox)
 table.insert(ee.widgets, box1)
-
--- register new module "external editors" in lighttable ------------------------
-if dt.gui.current_view().id == "lighttable" then
-  install_module()
```

Comment:
Have you tried starting darktable and having it open an image directly, I.e.darktable /path/to/an/image.xxx ?  The reason this is in there is that adding a module to lighttable, when you're not in llighttable view, crashes darktable.

Reply:
Indeed! 
I misunderstood the purpose of that part of code, sorry about that. 
Reverted now